code/modules/mining/mine_turfs.dm

```diff
@@ -619,10 +619,12 @@
 
 /turf/open/floor/plating/asteroid/basalt/lava_land_surface
 	initial_gas_mix = "o2=14;n2=23;TEMP=300"
+	planetary_atmos = TRUE
 	baseturf = /turf/open/floor/plating/lava/smooth/lava_land_surface
 
 /turf/open/chasm/straight_down/lava_land_surface
 	initial_gas_mix = "o2=14;n2=23;TEMP=300"
+	planetary_atmos = TRUE
```

Comment:
for the most part, i assumed the lava_land_surface suffix is only going to be used on the surface of the planet of lavaland.

`/turf/open/chasm/straight_down/lava_land_surface/normal_air` sounds like it should really be `/turf/open/chasm/straight_down/normal_air` what does the lava_land_surface child type give?